scripts/scripts.js

```diff
@@ -355,6 +355,10 @@ function decorateH3(main) {
   }
 }
 
+function updatePageTitle() {
+  document.title += ' - Service Now';
```

Comment:
Since this is super small and fast, I think we could simply add it here: https://github.com/hlxsites/servicenow/blob/main/scripts/scripts.js#L379 , after use set the document lang attribute.